bridge-adaptivity/bridge_lti/urls.py

```diff
@@ -2,12 +2,10 @@
 from django.contrib.auth.decorators import login_required
 
 from .provider import lti_launch
-from .views import LtiSourceList, LtiSourceDetail
-from .consumer import content_source
+from .consumer import source_preview
 
 urlpatterns = [
-    url(r'^launch/(?P<collection_id>\d+)$', lti_launch, name='lti-provider-launch'),
-    url(r'^sources/$', login_required(LtiSourceList.as_view()), name='source-list'),
-    url(r'^sources/(?P<pk>\d+)/$', login_required(LtiSourceDetail.as_view()), name='source-detail'),
-    url(r'^content_source/(?P<pk>\d+)/$', content_source, name='content-source'),
+    url(r'^launch/$', lti_launch, name='initial-launch'),
```

Comment:
It does not seem we need additional endpoint for initial launch we could just improve regex in launch endpoint

Reply:
do we want the initial launch to be as so 'host/lti/launch' (w/o ending slash)?